students/thecthaeh/Lesson02/series.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+    
+def fibonacci(n):
+    """Return the nth value in the fibonacci series."""
+    if n == 0:
+        return 0
+        
+    elif n == 1:
+        return 1
+        
+    return fibonacci(n - 2) + fibonacci(n - 1)    
+
+
+
+def lucas(n):
+    """ Return the nth value in the lucas series."""
+    if n == 0:
+        return 2
+        
+    elif n == 1:
+        return 1
+        
+    return lucas(n - 2) + lucas(n - 1)
+
+
+def sum_series(n, index0 = 0, index1 = 1):
```

Comment:
You'll want to remove the white space around the assignment operators here - PEP8 formatting